roles/grafana/templates/grafana.ini.j2

```diff
@@ -3,18 +3,22 @@
 # http://docs.grafana.org/installation/configuration
 # https://github.com/grafana/grafana/blob/master/conf/sample.ini
 
-{% for k, v in grafana_ini.items() %}
-{% if v is not mapping %}
-{{ k }} = {{ v }}
-{% endif %}
-{% endfor %}
-
```

Comment:
Wait. Why did you remove this?
This is required for sectionless configuration like `app_mode` and `instance_name`.

Without this block they cannot be set.

Please restore it.

https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#app_mode

Reply:
https://github.com/grafana/grafana-ansible-collection/pull/300/commits/28a75e1e98b97d0fa7f73d518788e64e8a49c013#r1894068877